articles/libraries/lock-android/v1/configuration.md

```diff
@@ -1,15 +1,15 @@
 ---
 description: Configuration options and methods for Lock for Android v1
 ---
-## Lock v1 Configuration
+# Lock v1 Configuration
```

Comment:
Why are the titles different? v1 is `Lock v1 Configuration` while v2 is `Lock Configuration Options`.
I'd expect to see `Lock Configuration Options` in both titles since the version is displayed directly after.